public/post-client.js

```diff
@@ -0,0 +1,159 @@
+$(document).ready(function(){
+
+  var Post = function(){
+    this.title;
+    this.content;
+    this.category;
+  };
+
+  Post.prototype.showAllPosts = function(){
+    $.ajax({
+      context: this,
+      type: 'GET',
+      url: '/posts',
+      success: function(response){
+        var html = '';
+
+        response.reverse().forEach(function (post) {
```

Comment:
you can tell mongo to sort however you want: http://docs.mongodb.org/manual/reference/operator/meta/orderby/